IBM_DB/ibm_db_django/ibm_db_django/base.py

```diff
@@ -268,7 +268,9 @@ def create_cursor( self ):
             return self.databaseWrapper._cursor( self.connection )
             
         def init_connection_state( self ):
-            pass
+            if ( self.settings_dict.keys() ).__contains__( 'SCHEMA' ):
```

Comment:
why not the more pythonic   `if 'SCHEMA' in self.settings_dict:`

Reply:
I was wondering that as well why it has been written with __contains__ elsewhere in the code.
See Line #L236

Even `( self )` let my eyes bleed :)
`base.py` just isn't very pretty.